drivers/sensor/adxl372/adxl372.h

```diff
@@ -319,10 +318,8 @@ struct adxl372_dev_config {
 	const char *spi_port;
 	u16_t spi_slave;
 	u32_t spi_max_frequency;
-#ifdef CONFIG_ADXL372_SPI_GPIO_CS
```

Comment:
This should be replaced with the new macro, not removed

Reply:
oh, missed that one.  Fixed now.